README.md

```diff
@@ -196,16 +148,11 @@ Example:
 
 ```
  -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
+👤 **Wasyihun Sema**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
```

Comment:
- [ ] While you did a wonderful job adding your name to the author's section :+1:, as a general requirement you should add your social media links.
**Hint:** you can write inside the parenthesis the link required and inside the brackets the text that you want to be shown :+1: